tests/unit/test_isort.py

```diff
@@ -5576,6 +5576,18 @@ def test_split_on_trailing_comma() -> None:
     assert output == expected_output
 
 
+def test_split_on_trailing_comma_wih_as() -> None:
+    test_input = "from lib import (a as b,)"
+    expected_output = """from lib import a as b
+"""
```

Comment:
what's expected for multiple `as` imports an a trailing comma, e.g. `from lib import (a as b, c as d,)?

```python
from lib import a as b
from lib import c as d
```

or

```python
from lib import (
    a as b
    c as d
)
```

Reply:
Current code will do:
```python
from lib import a as b
from lib import c as d
```